modules/javafx.controls/src/main/java/javafx/scene/control/TextFormatter.java

```diff
@@ -37,17 +37,18 @@
  * A Formatter describes a format of a {@code TextInputControl} text by using two distinct mechanisms:
  * <ul>
  *     <li>A filter ({@link #getFilter()}) that can intercept and modify user input. This helps to keep the text
- *     in the desired format. A default text supplier can be used to provide the intial text.</li>
+ *     in the desired format. A default text supplier can be used to provide the initial text.</li>
  *     <li>A value converter ({@link #getValueConverter()}) and value ({@link #valueProperty()})
  *     can be used to provide special format that represents a value of type {@code V}.
- *     If the control is editable and the text is changed by the user, the value is then updated to correspond to the text.
+ *     If the control is editable and the text is changed by the user, the value is then updated to correspond to the
+ *     text.
  * </ul>
  * <p>
- * It's possible to have a formatter with just filter or value converter. If value converter is not provided however, setting a value will
- * result in an {@code IllegalStateException} and the value is always null.
+ * It's possible to have a formatter with just filter or value converter. If value converter is not provided however,
```

Comment:
The "however" in the middle of the sentence is a bit awkward, and not really needed here. I suggest dropping it.